lib/rules/no-return-await.js

```diff
@@ -44,7 +46,32 @@ module.exports = {
             context.report({
                 node: context.getSourceCode().getFirstToken(node),
                 loc: node.loc,
-                messageId: "redundantUseOfAwait"
+                messageId: "redundantUseOfAwait",
+                suggest: [
+                    {
+                        messageId: "removeAwait",
+                        fix(fixer) {
+                            const areAwaitAndAwaitedExpressionOnTheSameLine = node.loc.start.line === node.argument.loc.start.line;
+
+                            if (!areAwaitAndAwaitedExpressionOnTheSameLine) {
+                                return null;
+                            }
+
+                            const sourceCodeText = context.getSourceCode().text;
+
+                            const startOfAwait = node.range[0];
+                            const endOfAwait = startOfAwait + "await".length;
+
+                            const characterAfterAwait = sourceCodeText.slice(endOfAwait, endOfAwait + 1);
```

Comment:
```suggestion
                            const characterAfterAwait = sourceCodeText[endOfAwait];
```

Since we need only 1 character, maybe we could simplify this?